Source/WebCore/page/ImageAnalysisQueue.h

```diff
@@ -51,6 +55,11 @@ class ImageAnalysisQueue {
 
     void enqueueIfNeeded(HTMLImageElement&);
 
+    WEBCORE_EXPORT void setImageQueueEmptyHysteresis(Function<void()>&& callback);
+    WEBCORE_EXPORT void endImageQueueEmptyHysteresis();
+
+    bool analysisOfAllImagesOnPageHasStarted() const { return m_analysisOfAllImagesOnPageHasStarted; }
```

Comment:
(You can then remove this method, if you roll the check into `enqueueAllImages`).